CooperationHullMainSite/Views/Home/Index.cshtml

```diff
@@ -154,17 +154,66 @@
+                        <p class="date-tag heading-xs">16th Nov</p>
+                        <img class="header-image" src="~/assets/img/pattern 2.jpeg" />
+
+                        <div class="content">
+                            <h3 class="heading-sm">HU4 Neighbourhood Assembly</h3>
+                            <div class="description-block">
+                                <p class="text-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eget dui tortor. Fusce blandit elit sit amet.</p>
+                                <a href='link' target='_blank'>
+                                    <img src="~/assets/img/next arrow.svg" />
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                
+                <div class="col-lg-4 col-md-6 col-sm-6 p-5">
+                    <div class="happening-next-item">
+                        <p class="date-tag heading-xs">16th Nov</p>
+                        <img class="header-image" src="~/assets/img/pattern 2.jpeg" />
+
+                        <div class="content">
+                            <h3 class="heading-sm">HU4 Neighbourhood Assembly</h3>
+                            <div class="description-block">
+                                <p class="text-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eget dui tortor. Fusce blandit elit sit amet.</p>
+                                <a href='link' target='_blank'>
+                                    <img src="~/assets/img/next arrow.svg" />
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
 
+                <div class="col-lg-4 col-md-6 col-sm-6 p-5">
+                    <div class="happening-next-item">
+                        <p class="date-tag heading-xs">16th Nov</p>
+                        <img class="header-image" src="~/assets/img/pattern 2.jpeg" />
+
+                        <div class="content">
+                            <h3 class="heading-sm">HU4 Neighbourhood Assembly</h3>
+                            <div class="description-block">
+                                <p class="text-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eget dui tortor. Fusce blandit elit sit amet.</p>
+                                <a href='link' target='_blank'>
+                                    <img src="~/assets/img/next arrow.svg" />
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                @* TODO: get data from eventList model *@
                 @foreach (HappenginNextEvent eventItem in Model.eventList)
                 {
                     @Html.Partial("_HappeningNext", eventItem)
-
                 }
```

Comment:
Need some dotnet wizard help to get this working, as its stumped me so far. As can be seen above, so far I've just hardcoded the example events.